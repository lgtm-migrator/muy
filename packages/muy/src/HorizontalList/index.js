import React from "react"
import makeStyles from "@material-ui/core/styles/makeStyles"
import List from "@material-ui/core/List"
import Box from "@material-ui/core/Box"

const useStyles = makeStyles({
    root: {
        "& li": {
            justifyContent: "space-around",
        },
    },
})

const HorizontalList = props => {
    const classes = useStyles()
    return (
        <Box
            classes={classes}
            component={List}
            disablePadding={true}
            display="flex"
            flexDirection="row"
            {...props}
        />
    )
}

export default HorizontalList
