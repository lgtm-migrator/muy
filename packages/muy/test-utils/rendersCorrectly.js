import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
