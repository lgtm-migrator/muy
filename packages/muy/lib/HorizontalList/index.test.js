import React from "react"
import rendersCorrectly from "../../test-utils/rendersCorrectly"
import HorizontalList from "."

describe("<HorizontalList />", () => {
  it("renders correctly with no target prop", () => {
    rendersCorrectly(<HorizontalList to={"/"}>Link</HorizontalList>)
  })
})
