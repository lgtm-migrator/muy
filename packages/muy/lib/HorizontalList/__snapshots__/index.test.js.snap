// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HorizontalList /> renders correctly with no target prop 1`] = `
<ul
  className="MuiList-root makeStyles-root-1 MuiBox-root MuiBox-root-2 makeStyles-root-1"
  to="/"
>
  Link
</ul>
`;
